Program.cs

```diff
@@ -0,0 +1,113 @@
+﻿using OpenQA.Selenium;
+using Mars.Drivers;
+using Mars.Helpers;
+using System;
+using dotenv.net;
+
+namespace Mars
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            // Load environment variables from .env file
+            DotEnv.Load();
+
+            // Initialize WebDriver
+            IWebDriver driver = WebDriverInitializer.Initialize();
+
+            // Set up logging
+            Logger.InitializeLogFile();
+
+            // Perform login
+            Logger.Log("Logging into MARS...");
+            AuthenticationHelper.Login(driver);
+            Logger.Log("User successfully logged in.");
+
+            // Cleanup Task - Remove all existing languages and skills
+            Logger.Log("Starting cleanup of existing languages and skills...");
+            CleanupHelper.CleanupLanguagesAndSkills(driver);
+            Logger.Log("Cleanup completed successfully.");
+
+            // Handle Languages Section
+            RunLanguageTests(driver);
+
+            // Handle Skills Section
+            RunSkillTests(driver);
+
+            // Optionally quit the driver after the tests are done
+            driver.Quit();
+
+            // Final log message
+            Logger.Log("Test suite completed successfully.");
+        }
+
+        public static void RunLanguageTests(IWebDriver driver)
+        {
+            LanguageSteps languageSteps = new LanguageSteps(driver);
```

Comment:
also your LanguageSteps are not pushed up can you please push all your code thanks

Reply:
i dont see your language and skill pages